Library/Formula/supervisor.rb

```diff
@@ -0,0 +1,133 @@
+
+    touch libexec/"lib/python2.7/site-packages/supervisor/__init__.py"
+    bin.install Dir[libexec/"bin/supervisor*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+
+    (prefix/"supervisord.conf").write supervisord_conf
+  end
+
+  def post_install
+    (etc/"supervisord/conf.d").mkpath unless File.directory? etc/"supervisord/conf.d"
+    cp prefix+"supervisord.conf", etc unless File.exist? etc/"supervisord.conf"
+  end
+
+  def caveats; <<-EOS.undent
+    The supervisorctl command will need to be run with the full path to the conf file:
+        supervisorctl -c #{etc}/supervisord.conf
+    You can alias the command in your shell to make it easier:
+        alias supervisorctl='supervisorctl -c #{etc}/supervisord.conf'
+    or with root access link the config to /etc:
+        ln -s #{etc}/supervisord.conf /etc/supervisord.conf
+
+    Add your supervisor program config to: (http://supervisord.org/subprocess.html)
+        #{etc}/supervisord/conf.d/
+    EOS
+  end
+
+  test do
+    assert_match(/#{version}/, shell_output("#{bin}/supervisord --version"))
+    assert_match(/\nOptions:/, shell_output("#{bin}/supervisorctl --help"))
+  end
+
+  def supervisord_conf; <<-EOS.undent
+    [unix_http_server]
+    file=#{var}/run/supervisor.sock
+    chmod       = 0700
+    ;username    = admin
+    ;password    = somepass
+
+    ;[inet_http_server]
+    ;port        = 127.0.0.1:9001
+    ;username    = admin
+    ;password    = somepass
+
+    [supervisord]
+    logfile     = #{var}/log/supervisord.log
+    logfile_maxbytes = 50MB
+    logfile_backups = 10
+    loglevel    = info
+    pidfile     = #{var}/run/supervisord.pid
+    nodaemon    = false
+    minfds      = 1024
+    minprocs    = 200
+    umask       = 022
+    identifier   = supervisor
+    ;directory   = #{var}/run
+    ;nocleanup   = true
+    ;childlogdir = #{var}/log
+
+    [supervisorctl]
+    serverurl   = unix://#{var}/run/supervisor.sock
+    ;serverurl   = http://127.0.0.1:9001
+    ;username    = admin
+    ;password    = somepass
+
+    [rpcinterface:supervisor]
+    supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
+
+    [include]
+    files = #{etc}/supervisord/conf.d/*.conf
+    EOS
+  end
+
+  # vars don't seem to be available here
+  plist_options :manual => "supervisord -c #{HOMEBREW_PREFIX}/etc/supervisord.conf"
+
+  def plist; <<-EOS.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+    <dict>
+        <key>KeepAlive</key>
+        <dict>
+            <key>SuccessfulExit</key>
+            <false/>
+        </dict>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>ProgramArguments</key>
+        <array>
+            <string>#{opt_bin}/supervisord</string>
+            <string>-n</string>
```

Comment:
The config file generated above has `nodaemon = false` (daemonize) but this overrides it with `-n` (no daemonize).  The config file could just use `nodaemon = true`.